js/components/QuestionTypes/ScaleQuestion.js

```diff
@@ -0,0 +1,92 @@
+import React, {
+  StyleSheet,
+  Text,
+  TextInput,
+  View,
+  Slider,
+} from 'react-native'
+import Styles from '../../styles/Styles'
+
+const ScaleQuestion = React.createClass ({
+  propTypes: {
+    id: React.PropTypes.string.isRequired,
+    text: React.PropTypes.string.isRequired,
+    value: React.PropTypes.number,
+    onChange: React.PropTypes.func.isRequired,
+    properties: React.PropTypes.shape({
+      min: React.PropTypes.number.isRequired,
+      max: React.PropTypes.number.isRequired,
+      minText: React.PropTypes.text,
+      maxText: React.PropTypes.text,
+    }),
+  },
+
+  getDefaultProps: function () {
+    return {
+      value: 1,
+      properties: {
+        min: 0,
+        max: 5,
+      },
+    };
+  },
+
+  getInitialState: function() {
+    return {
+      value: this.props.value,
+    }
+  },
+
+  /* Methods */
+  handleChange(value) {
+    this.setState({
+      value: value
+    })
+    this.props.onChange(value)
+  },
+
+  /* Render */
+  renderNotes() {
```

Comment:
Called by one of the Slider component props "onValueChange".
Triggers automatically when the user uses the slider.